assets/subworkflows/espresso/calc_valence_band_offset.yml

```diff
@@ -0,0 +1,31 @@
+application:
```

Comment:
Maybe `valence_band_offset_calc_from_esp_vbm`, having first the property then the details

Reply:
We should add a quick description at the top and state that this expects the other subworkflow to interface with and is not a self-sufficient subworkflow per se